src/java/picard/illumina/quality/CollectHiSeqXPfFailMetrics.java

```diff
@@ -58,12 +58,65 @@
  * @author Yossi Farjoun
  */
 @CommandLineProgramProperties(
-        usage = "Classify PF-Failing reads in a HiSeqX Illumina Basecalling directory into various categories. The classification is based on a heuristic that was derived by looking at a few titration experiments.",
-        usageShort = "Classify PF-Failing reads in a HiSeqX Illumina Basecalling directory into various categories.",
+        usage = CollectHiSeqXPfFailMetrics.USAGE_SUMMARY + CollectHiSeqXPfFailMetrics.USAGE_DETAILS,
+        usageShort = CollectHiSeqXPfFailMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 
 public class CollectHiSeqXPfFailMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Classify PF-Failing reads in a HiSeqX Illumina Basecalling directory into various categories.";
+    static final String USAGE_DETAILS = "<p>This tool categorizes the reads that did not pass filter (PF-Failing) into four " +
+            "groups.  These groups are based on a heuristic that was derived by looking at a few titration experiments. </p>" +
+            "" +
+            "<p>After examining the called bases from the first 24 cycles of each read, the PF-Failed reads are grouped into the " +
+            "following four categories: " +
+            "<ul>" +
+            "<li>MISALIGNED - The first 24 basecalls of a read are uncalled (numNs~24).  " +
+            " These types of reads appear to be flow cell artifacts because reads " +
+            "were only found near tile boundaries and were concentration (library) independent</li> " +
+            "<li>Empty wells - All 24 bases are called (numNs~0) but the number of bases with quality scores" +
+            " greater than two is less than or equal to eight (numQGtTwo<=8).  These reads were location independent within the tiles and " +
+            "were inversely proportional to the library concentration</li>" +
+            "" +
+            "<li>Polyclonal wells - All 24 bases were called and numQGtTwo>=12, were independent of their location with the tiles, " +
+            "and were directly proportional to the library concentration.  These reads are likely the result of PCR artifacts </li>" +
+            "<li>UNKNOWN - The remaining reads that PF-Failing but did not fit into any of the groups listed above</li>"   +
```

Comment:
Changed to "The remaining reads that are PF-Failing but did not fit into any of the groups listed above."  Not sure if this is overusing the abbreviation.